STYLING.md

```diff
@@ -0,0 +1,225 @@
+# React Native Styling Guidelines
+
+## Where to Define Styles
+
+All styles must be defined in the `styles.js` file which exists as a globally exported stylesheet of sorts for the application. Unlike some React Native applications we are not using `StyleSheet.create()` and instead store styles as plain JS objects. There are many helper styles available for direct use in components and can be found in the `/styles` directory.
+
+These helper styles are loosely based on the [Bootstrap system of CSS utility helper classes](https://getbootstrap.com/docs/5.0/utilities/spacing/) and are typically incremented by units of 4.
+
+**Note:** Not all helpers from Bootstrap exist, so it may be necessary to create the helper style we need.
+
+## When to Create a New Style
+
+If we need some minimal set of styling rules applied to a component then it's almost always better to use an array of helper styles rather than create an entirely new style if it will only be used once. Resist the urge to create a new style for any new element added to a screen. There is a very good chance the style we are adding is a "single-use" style.
+
+```jsx
+// Bad - Since we only use this style once in this component
+const TextWithPadding = props => (
+    <Text style={styles.textWithPadding}>
+        {props.children}
+    </Text>
+);
+
+// Good
+const TextWithPadding = props => (
+    <Text
+        style={[
+            styles.p5,
+            styles.noWrap,
+        ]}
+    >
+        {props.children}
+    </Text>
+);
+```
+
+On the other hand, if we are copying and pasting some chunks of JSX from one place to another then that might be a sign that we need either a new component or a new reusable style.
+
+## Use the "Rule of Three"
+
+Any array of styles associated with a single type of component or element that has at least 3 identical usages should be refactored into either:
+
+- A new component with that same array of helper styles
+
+- A new style that is a composite of all helper styles
+
+## Inline Styles
+
+**Inline styles are forbidden.** If we run into a case where we feel it's necessary to conditionally render some styles. There are two options:
+
+1. Create a helper function in `styles.js` and pass any modifying parameters to that function.
+1. Conditionally render a style object inline by defaulting to `undefined`.
```

Comment:
We could probably also just settle on one of these options so that the guidance is clearer when this pops up. I think the helper function style makes the most sense.

Reply:
I'm happy to settle on just using helper methods, but I'm not opposed to leaving both methods for now (I'm not sure how much this happens).

Reply:
Not too often but often enough that we will have tons eventually. But yeah I think it's fine to promote the helper method for now and will simplify the guidance a bit.